hack/make-rules/test-cmd.sh

```diff
@@ -2313,8 +2313,9 @@ __EOF__
   kubectl-with-retry rollout resume deployment nginx "${kube_flags[@]}"
   # The resumed deployment can now be rolled back
   kubectl rollout undo deployment nginx "${kube_flags[@]}"
-  # Check that the new replica set (nginx-618515232) has all old revisions stored in an annotation
-  kubectl get rs nginx-618515232 -o yaml | grep "deployment.kubernetes.io/revision-history: 1,3"
```

Comment:
Is the change to use versioned api going to change hashes in all pre-existing deployments? There was value in this test in that we ensured that the hash wouldn't change between updates. As a matter of fact we [need such a test](https://github.com/kubernetes/kubernetes/issues/40415) and I created one in https://github.com/kubernetes/kubernetes/pull/40854.